realm/realm-library/src/main/java/io/realm/util/Pair.java

```diff
@@ -14,19 +14,22 @@
  * limitations under the License.
  */
 
-package io.realm.internal.util;
+package io.realm.util;
```

Comment:
hmm? But i cannot see `io.realm.util` package here https://realm.io/docs/java/3.5.0/api/

UH... my fault, there is a `io.realm.util` in the `androidTest`. This is OK then 👍